parsec/utils/output.c

```diff
@@ -568,13 +568,16 @@ static int do_open(int output_id, parsec_output_stream_t * lds)
             parsec_atomic_unlock(&mutex);
             return PARSEC_ERR_OUT_OF_RESOURCE;
         }
+        info[i].ldi_used = true;
+        parsec_atomic_unlock(&mutex);
     }
 
     /* Otherwise, we're reopening, so we need to free all previous
      * resources, close files, etc. */
 
     else {
         free_descriptor(output_id);
+        info[output_id].ldi_used = true;
```

Comment:
`=false` ?

Reply:
No. `.ldi_used` is true when the location is a valid output stream. In this else branch we are reinitializing an output stream using `free_descriptor()`, which sets the location to unused. So we need to flip it back to used.